### PR TITLE
Created Navbar

### LINKED ISSUE #5: NavBar
image

### PR DESCRIPTION
**Pull request by wx02shi**
closing #5 
Added global padding (8% width padding on the sides)
Created navbar
logo added to public folder
Not sure if Montserrat font will stay

### CODE REVIEW

**Comment by functionstackx** [comment]
@williamsxz99 can you put a screenshot in the description of this PR of what you built

**Comment by wx02shi** [comment]
image

**Comment by functionstackx** [comment]
i know you new to git workflow.
you wanna add a reviewer, and put yr self as a assignee.then add a project and a milestone. and use the #closing followed by the issue number.

so in this case, you need to do like "  closing #5    " in the description of the PR

image

**Comment by wx02shi** [comment]
okay. So assigning myself in the project section doesn't assign it here right?

**Comment by functionstackx** [comment]
you assigned yrself to the issue. you need to assign yrself to the PR.

**Comment by wx02shi** [comment]
ah right okay. Do I "request" reviewers when I make a PR?

**Comment by functionstackx** [comment]
yeet, you need to fill out all of these on the right side of the PR

image

**Comment by wx02shi** [comment]
kk

**Comment by functionstackx** [comment]
Here is an example of how to fill it out.

you click the setting clog button
image


https://github.com/Cache-and-Cookies/msmr/pull/27

image

**Comment by wx02shi** [comment]
Ok, I've updated this one with the related changes

File: styles/globals.css [hunk 1]
```diff
@@ -1,8 +1,10 @@
-html,
 body {
-  padding: 0;
+  padding-left: 8%;
+  padding-right: 8%;
+  padding-top: 70px;
   margin: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Montserrat', sans-serif !important;
```
Comment (functionstackx) [inline styles/globals.css:6]: what was the use of !important here?

Comment (wx02shi) [reply styles/globals.css:6]: The Montserrat font wasn't actually showing until I put it in

Comment (functionstackx) [reply styles/globals.css:6]: ahh ok, we can investigate this later

**Review by functionstackx — APPROVED** [review]

**Pull request merged**